modal/_runtime/container_io_manager.py

```diff
@@ -452,7 +452,7 @@ async def _dynamic_concurrency_loop(self):
             await asyncio.sleep(DYNAMIC_CONCURRENCY_INTERVAL_SECS)
 
     async def get_app_objects(self) -> RunningApp:
-        req = api_pb2.AppGetObjectsRequest(app_id=self.app_id, include_unindexed=True)
+        req = api_pb2.AppGetObjectsRequest(app_id=self.app_id, include_unindexed=True, only_class_function=True)
```

Comment:
Do we have a comment explaining the purpose / limited lifetime of `only_class_function` somewhere? I didn't see it in api.proto. (In general we're not very good about documenting the protos, but should try to get better).